data-model/src/main/scala/za/co/absa/enceladus/model/Dataset.scala

```diff
@@ -71,8 +71,7 @@ case class Dataset(name: String,
   def setSchemaVersion(newVersion: Int): Dataset = this.copy(schemaVersion = newVersion)
   def setHDFSPath(newPath: String): Dataset = this.copy(hdfsPath = newPath)
   def setHDFSPublishPath(newPublishPath: String): Dataset = this.copy(hdfsPublishPath = newPublishPath)
-  def setConformance(newConformance: Option[List[ConformanceRule]]): Dataset = this.copy(conformance = newConformance.getOrElse(Nil))
-  def setSchedule(newSchedule: Option[OozieSchedule]): Dataset = this.copy(schedule = newSchedule)
```

Comment:
(just a question for my own benefit)

I can see that you removed `setSchedule` method - okay, I can see that it's not used anywhere in the repo. However, what about the `schedule` attribute itself? I assume that for backward compatibility we want to keep it there - as well as `OozieSchedule` class and a few other usages across the codebase. If that's the case, can you just put a small comment above the attribute `schedule` about this?

What do you think, @benedeki?

Reply:
You are right @lsulak , the fields are kept in the model for compatibility (was planned to be removed with new data model).
Comment would be good. Please add it @TebaleloS

Reply:
Okay cool.